.github/workflows/release.yml

```
@@ -86,31 +101,129 @@ jobs:
+          SPARKLE_VERSION="2.8.0"
+          SPARKLE_URL="https://github.com/sparkle-project/Sparkle/releases/download/${SPARKLE_VERSION}/Sparkle-${SPARKLE_VERSION}.tar.xz"
+          echo "Downloading Sparkle tools from: $SPARKLE_URL"
+          curl -L "$SPARKLE_URL" | tar -xJ
+          if [ ! -x "./bin/sign_update" ]; then
+            echo "❌ sign_update binary not found or not executable"
+            exit 1
+          fi
+          echo "✅ Sparkle tools downloaded and extracted"
+
+      - name: Sign Sparkle update
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2064:warning:16:14: Use single quotes, otherwise this expands now rather than when signalled

(shellcheck)